src/brs/at/AtController.java

```diff
@@ -517,35 +519,51 @@ private static int getCostOfOneAT() {
 
   //platform based implementations
   //platform based
-  private static long makeTransactions(AT at, int blockHeight, long generatorId) throws AtException {
-    long totalAmount = 0;
-    if (!Signum.getFluxCapacitor().getValue(FluxValues.AT_FIX_BLOCK_4, at.getHeight())) {
-      for (AtTransaction tx : at.getTransactions()) {
-        if (AT.findPendingTransaction(tx.getRecipientId(), blockHeight, generatorId)) {
-          throw new AtException("Conflicting transaction found");
+    private static long makeTransactions(AT at, int blockHeight, long generatorId) throws AtException {
+        long totalAmount = 0;
+
+        // Sort transactions by priority: MINT first, then TRANSFER, etc.
```

Comment:
```suggestion
```
don't comment what is obvious,... the Prio is implementation detail and inside comparator